src/classmentors/components/cohorts/cohorts.js

```diff
@@ -583,6 +582,7 @@ function EditCohortCtrl(initialData, spfNavBarService, urlFor, spfAlert, clmData
     this.querySearch = querySearch;
 
     function querySearch (query) {
+        console.log(self.mappedEvents);
```

Comment:
Please use $log if it's meant to be permanent. Try to avoid committing console calls